configure.ac

```diff
@@ -1,152 +1,213 @@
 #                                               -*- Autoconf -*-
 # Process this file with autoconf to produce a configure script.
 
-AC_PREREQ(2.65)
-AC_INIT([htop],[3.0.6-dev],[htop@groups.io])
+# ----------------------------------------------------------------------
+# Autoconf initialization.
+# ----------------------------------------------------------------------
+
+AC_PREREQ([2.69])
+AC_INIT([htop], [3.0.6-dev], [htop@groups.io], [], [https://htop.dev/])
 
 AC_CONFIG_SRCDIR([htop.c])
-AC_CONFIG_AUX_DIR([.])
+AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_HEADERS([config.h])
 
-# Required by hwloc scripts
 AC_CANONICAL_TARGET
+AM_INIT_AUTOMAKE([-Wall std-options subdir-objects])
 
-AM_INIT_AUTOMAKE([1.11])
```

Comment:
What's the differnce between those two invocations? What do they accomplish?

Reply:
See https://www.gnu.org/software/automake/manual/html_node/Public-Macros.html for details:
Passing a version is deprecated.
See https://www.gnu.org/software/automake/manual/html_node/List-of-Automake-options.html for details on the options:
 - `-Wall`: enable automake warnings
 - `std-options`: check generated executables support `--help` and `--version`
 - `subdir-objects`: generate object files in subdirs, e.g. *linux/LinuxPorcess.o* instead of *./LinuxProcess.o*